aws.is-local.org.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "AWS TEST Project",
+
+    "domain": "is-local.org",
+    "subdomain": "aws",
+
+    "owner": {
+        "repo": "https://github.com/afom/register",
+        "email": "shino@ifnet.or.jp"
+    },
+
+    "record": {
+        "NS": ["NS1.VALUE-DOMAIN.COM", "NS2.VALUE-DOMAIN.COM", "NS3.VALUE-DOMAIN.COM", "NS4.VALUE-DOMAIN.COM", "NS5.VALUE-DOMAIN.COM"],
```

Comment:
Please provide a detailed explanation on why you need NS records.

Reply:
Because we want to test the delegation of the subdomain aws.is-local.org that we have acquired here.

Reply:
Sounds good. I'm leaving a comment on txt record.

Reply:
I removed txt record.